### PR TITLE
Add numbers to faction member list

### LINKED ISSUE #339: Show the position of the faction member in the list.
**Describe the solution you'd like**
Show the position of the faction member in the list.

Bad Example:
Bad Example

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #339 .

### CODE REVIEW

File: torntools/scripts/content/faction/ttFaction.css [hunk 1]
```diff
@@ -150,6 +150,11 @@
 	margin-bottom: 10px;
 }
 
+.members-list .table-body .tt-member-index {
+	margin-right: -7px;
+    padding: 10px 5px 0px 5px;
```
Comment (DeKleineKobini) [inline torntools/scripts/content/faction/ttFaction.css:155]: 0 instead of 0px

File: torntools/scripts/content/faction/ttFaction.js [hunk 2]
```diff
@@ -1780,3 +1781,11 @@ function exportChallengeContributionsCSV() {
 		contributionsObserver.observe(doc.find("div#factions div#faction-upgrades div.body div#stu-confirmation"), { childList: true, subtree: true });
 	});
 }
+
+function addNumbersToMembers() {
+	let memberIndex = 1;
+	doc.findAll("div#faction-info-members ul.table-body li.table-row").forEach((memberRow) => {
```
Comment (DeKleineKobini) [inline torntools/scripts/content/faction/ttFaction.js:1787]: `(row, index) =>` that way you don't have to update the index yourself, just show `index + 1` then.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
I love it, but there are 2 things missing:
- changelog, as usual xD
- a setting for, although it can be on by default

File: torntools/scripts/content/faction/ttFaction.js (hunk 2 above)
Comment (Sashank999) [reply torntools/scripts/content/faction/ttFaction.js]: Done.

File: torntools/scripts/content/faction/ttFaction.css (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/faction/ttFaction.css]: Done.

**Comment by Sashank999** [comment]
Changelog done.

**Comment by Sashank999** [comment]
Setting is added.

**Pull request merged**